img_proof/ipa_distro.py

```diff
@@ -170,3 +170,22 @@ def update(self, client):
                 'An error occurred updating instance: %s' % error
             )
         return out
+
+    def refresh(self, client):
```

Comment:
I think repo_refresh would make things a bit more explicit and obvious for the reader and set proper expectations.